exonum/src/node/connect_list.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Module that provides methods for managing the mapping between peers
```

Comment:
I don't like such (module/structure/method that...) phrasing. It is obvious that you are describing the module (structure or method). Can we say it somehow different?

For example, "Utilities for managing the..." or even simply "Mapping between...".